pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -176,15 +176,15 @@ func (m *managerImpl) Admit(attrs *lifecycle.PodAdmitAttributes) lifecycle.PodAd
 // Start starts the control loop to observe and response to low compute resources.
 func (m *managerImpl) Start(diskInfoProvider DiskInfoProvider, podFunc ActivePodsFunc, podCleanedUpFunc PodCleanedUpFunc, monitoringInterval time.Duration) {
 	thresholdHandler := func(message string) {
-		klog.Infof(message)
+		klog.InfoS(message)
 		m.synchronize(diskInfoProvider, podFunc)
 	}
 	if m.config.KernelMemcgNotification {
 		for _, threshold := range m.config.Thresholds {
 			if threshold.Signal == evictionapi.SignalMemoryAvailable || threshold.Signal == evictionapi.SignalAllocatableMemoryAvailable {
 				notifier, err := NewMemoryThresholdNotifier(threshold, m.config.PodCgroupRoot, &CgroupNotifierFactory{}, thresholdHandler)
 				if err != nil {
-					klog.Warningf("eviction manager: failed to create memory threshold notifier: %v", err)
+					klog.InfoS("Eviction manager: failed to create memory threshold notifier", "err", err)
```

Comment:
why warning is replaced with Informational?

Reply:
I modified it according to the following rules
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md

Reply:
hm, interesting. thank you!